http-tests/start-local.sh

```
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+# Miche-Let 로컬 서비스 기동 스크립트
+# 사용법: ./start-local.sh [서비스명]
+# 예시:   ./start-local.sh user-service
+#         ./start-local.sh all  (모든 서비스 순서대로 기동)
+
+set -e
+
+# infra .env 로드
+ENV_FILE="$(dirname "$0")/infra/.env"
+if [ -f "$ENV_FILE" ]; then
+  export $(grep -v '^#' "$ENV_FILE" | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)